modules/cloudfront.js

```diff
@@ -162,6 +162,7 @@ function generateBehaviours(origins, originName) {
      * or custom cookies to be whitelisted you must define those rules here.
      */
     let customPaths = [
+        { path: '/-/media/files/*', isPostable: false, allowAllQueryStrings: true },
```

Comment:
Not 100% sure why we need this route in here? Also it's likely going to clash with the `/-/*` path rule above which sends anything under that to the legacy servers.

https://github.com/biglotteryfund/blf-alpha/blob/b412017bc281c0136f672f3d9f4f7e9855b3de2f/modules/cloudfront.js#L123-L124

Reply:
This is where the original code was adding this path before my change...? I just took out the `if` statement.

Reply:
This is how it looks on TEST currently, based on the same config:

![image](https://user-images.githubusercontent.com/394376/50961137-e59b1b00-14be-11e9-8982-318472da7f52.png)

Reply:
... so yeah, you're right, the first rule takes precedence. Will have a fiddle.